pyproject.toml

```diff
@@ -40,19 +40,20 @@ classifiers = [
 ]
 
 dependencies = [
-    "torchrl>=0.2.0",
-    "tensordict>=0.2.0",
-    "lightning>=2.1.0",
+    "einops",
     "hydra-core",
     "hydra-colorlog",
+    "lightning>=2.1.0",
+    "matplotlib",
     "omegaconf",
     "pyrootutils",
     "rich",
-    "einops",
-    "wandb",
-    "matplotlib",
+    "robust-downloader",
     "scipy",
-    "robust-downloader"
+    "tensordict>=0.2.0",
+    "torchrl>=0.2.0",
+    "vrplib>=1.3.0",
```

Comment:
(minor) should we include problem-specific dependencies as requirements, or is it better to have them as `optional-dependencies`? 
CC @cbhua